tests/Integration/Form/Activity/ActivityCreationFlowTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Tests\Form\Security\Import;
+
+use App\Form\Activity\ActivityCreationFlow;
+use App\Form\Activity\ActivityLocationType;
+use App\Form\Activity\ActivityNewType;
+use PHPUnit\Framework\TestCase;
+
+class ActivityCreationFlowTest extends TestCase
+{
+    public function testLoadStepsConfig(): void
```

Comment:
Although I see that the structure of this test resembles the existing tests of flows, I'd propose we'd test flows the same way we test form: by evaluating the resulting flow structures, instead of the raw builder-type content within the unit itself.